<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMainWindow</class>
 <widget class="QWidget" name="AdminMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>573</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1012</width>
    <height>573</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1012</width>
    <height>573</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HomePage Admin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0082b6;
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>89</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <property name="title">
    <string>Cerca un Utente</string>
   </property>
   <widget class="QLineEdit" name="nameFindLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surnameFindLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameFindLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color:white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Cognome</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nameRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="usernameRadioButton">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>50</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>189</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <property name="title">
    <string>Elimina un Utente</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameDeleteLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color:white;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>351</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <property name="title">
    <string>Cambia tipologia account ad un utente</string>
   </property>
   <widget class="QLineEdit" name="usernameChangeLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color:white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="basicRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(78, 155, 209);
border:none;
color:white;</string>
     </property>
     <property name="text">
      <string>Basic</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="businessRadioButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(78, 155, 209);
border:none;
color:white;</string>
     </property>
     <property name="text">
      <string>Business</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="executiveRadioButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(78, 155, 209);
border:none;
color:white;</string>
     </property>
     <property name="text">
      <string>Executive</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <property name="title">
    <string>Inserisci nuovo Utente</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="updateTextBrowser">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color:black;
border:1px solid;
border-radius: 10px 10px 10px 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>HomePage Amministratore</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1021</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>510</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Chiudi</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>470</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <property name="title">
    <string>Salvataggio su File del database</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FindUserButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color:white;</string>
   </property>
   <property name="text">
    <string>Cerca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteUserButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color:white;</string>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeAccountUserButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color:white;</string>
   </property>
   <property name="text">
    <string>Cambia</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterNewUser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color:black;</string>
   </property>
   <property name="text">
    <string>Nuovo Utente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveDB">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>510</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color:black;</string>
   </property>
   <property name="text">
    <string>Salva DB</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="resultTable">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>431</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cognome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Username    </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TipoUtente</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>470</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <property name="title">
    <string>Eliminazione totale database</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emptyDbButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>505</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color:black;</string>
   </property>
   <property name="text">
    <string>Empty DB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getAllUsers">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color:white;</string>
   </property>
   <property name="text">
    <string>Visualizza tutti gli utenti</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>831</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Per applicare e salvare tutti i cambiamenti nel file database XML, premere il pulsante dedicato (Salva DB) prima di uscire dalla finestra.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
